src/pages/About/index.tsx

```diff
@@ -1,16 +1,16 @@
 import Button from "@material-ui/core/Button";
-import { motion } from "framer-motion";
-import { useHistory } from "react-router";
-import { SignIn } from "../../components/SignIn";
-import { useUserState } from "../../store/user/useUserState";
-import "./About.css";
-import { makeStyles } from "@material-ui/core/styles";
 import Card from "@material-ui/core/Card";
 import CardActionArea from "@material-ui/core/CardActionArea";
 import CardActions from "@material-ui/core/CardActions";
 import CardContent from "@material-ui/core/CardContent";
 import CardMedia from "@material-ui/core/CardMedia";
+import { makeStyles } from "@material-ui/core/styles";
 import Typography from "@material-ui/core/Typography";
+import { motion } from "framer-motion";
+import { useHistory } from "react-router";
+import { SignIn } from "../../components/SignIn";
+import { useUserState } from "../../store/user/useUserState";
+import "./About.css";
```

Comment:
If you can change for this component to use styles as CSS modules 🙏

Reply:
Updated to use CSS module!